<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_calc">
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="text_objFun">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPlainTextEdit" name="text_conFun"/>
    </item>
    <item row="3" column="0">
     <widget class="QTextBrowser" name="disp_logger"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>P&amp;lik</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionWczytaj"/>
    <addaction name="separator"/>
    <addaction name="actionZapisz"/>
    <addaction name="actionZapisz_jako"/>
    <addaction name="separator"/>
    <addaction name="actionZamknij"/>
   </widget>
   <widget class="QMenu" name="menuOpcje">
    <property name="title">
     <string>Op&amp;cje</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionTw_rcy"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuOpcje"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionWczytaj">
   <property name="text">
    <string>&amp;Wczytaj</string>
   </property>
  </action>
  <action name="actionZapisz">
   <property name="text">
    <string>&amp;Zapisz</string>
   </property>
  </action>
  <action name="actionZapisz_jako">
   <property name="text">
    <string>Zapisz &amp;jako</string>
   </property>
  </action>
  <action name="actionZamknij">
   <property name="text">
    <string>Z&amp;amknij</string>
   </property>
  </action>
  <action name="actionTw_rcy">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/new/images/hammer.png</normaloff>:/new/images/hammer.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Tw√≥rcy</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
